iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExampleOptions.java

```diff
@@ -266,10 +230,11 @@ public static DeviceRegistryExampleOptions fromFlags(String[] args) {
       String footer = "\nhttps://cloud.google.com/iot-core";
 
       HelpFormatter formatter = new HelpFormatter();
-      formatter.printHelp("DeviceRegistryExample", header, options, footer, true);
+      formatter.printHelp("DeviceRegistryExample", header, options, footer,
```

Comment:
Please move the new line before the start of arguments instead of part way through.

Reply:
Okay 👍